zerver/openapi/zulip.yaml

```diff
@@ -12821,6 +12836,24 @@ paths:
                           - 2 = Nobody
 
                           **Changes**: New in Zulip 3.0 (feature level 1).
+                      realm_direct_message_initiator_group_id:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The ID of the user group in the organization which has the permission
+                          to initiate direct message threads.
+
+                          **Changes**: New in Zulip 7.0 (feature level 182).
+                      realm_direct_message_permission_group_id:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          The ID of the user group in the organization authorized to participate
+                          in direct message threads as sender or recipient.
```

Comment:
Let's spell this one out a bit more too, adding "Direct messages can only been sent in direct message groups where at least one of the participating users is in this group. Setting this option to the Nobody system group will result in direct messages between users being disabled in the organization. Users can always send direct messages to themselves and to bots regardless of this setting."